packages/app/src/app/pages/Dashboard/Components/WhatsNew/WhatsNew.tsx

```diff
@@ -0,0 +1,149 @@
+  Grid,
+  Column,
+  Icon,
+  Link,
+  IconButton,
+} from '@codesandbox/components';
+
+interface WhatsNewProps {
+  onClose: () => void;
+}
+
+export const WhatsNew = ({ onClose }: WhatsNewProps) => {
+  return (
+    <Media query="(max-width: 549px)">
+      {matchSmall => (
+        <Media query="(min-width: 550px) and (max-width: 689px)">
+          {matchMedium => (
+            <>
+              <Element paddingBottom={matchSmall ? 20 : 80}>
+                <Stack
+                  direction="horizontal"
+                  justify="space-between"
+                  css={css({ padding: '24px' })}
+                >
+                  <Text
+                    variant="muted"
+                    size={13}
+                    css={css({ lineHeight: '16px' })}
+                  >
+                    What&apos;s new
+                  </Text>
+                  <IconButton
+                    name="cross"
+                    size={16}
+                    title="Close modal"
+                    onClick={onClose}
+                  />
+                </Stack>
+              </Element>
+
+              <Element
+                paddingX={matchSmall ? 24 : 48}
+                paddingBottom={matchSmall ? 44 : 56}
+              >
+                <Stack direction="vertical" gap={6}>
+                  <Text
+                    size={12}
+                    weight="500"
+                    css={css({
+                      lineHeight: '20px',
+                      alignSelf: 'flex-start',
+                      backgroundColor: '#EDFFA5',
+                      color: '#0E0E0E',
+                      padding: '0px 6px',
+                      borderRadius: '3px',
+                    })}
+                  >
+                    New
+                  </Text>
+                  <Text
+                    as="h2"
+                    size={matchSmall ? 44 : 64}
+                    weight="500"
+                    css={css({
+                      margin: 0,
+                      fontFamily: 'Everett, sans-serif',
+                      lineHeight: matchSmall ? '44px' : '64px',
+                      letterSpacing: '-0.03em',
+                    })}
+                  >
+                    Introducing a unified CodeSandbox experience.
+                  </Text>
+                </Stack>
+              </Element>
+
+              <Element
+                paddingX={matchSmall ? 24 : 48}
+                paddingBottom={matchSmall ? 44 : 56}
+              >
+                <Grid columnGap={5}>
+                  <Column span={matchSmall || matchMedium ? 6 : 3}>
+                    <Topic icon="repository">
+                      Projects is now <Text weight="700">Repositories</Text>: an
+                      improved git workflow powered by the cloud.
+                    </Topic>
+                  </Column>
+
+                  <Column span={matchSmall || matchMedium ? 6 : 3}>
+                    <Topic icon="dashboard">
```

Comment:
This isn't the house icon, but we don't use it yet in the client codebase. Is this OK for the dashboard icon?

Reply:
Fixed it and we're now using the right icon.